v0/destinations/google_adwords_remarketing_lists/util.js

```diff
@@ -100,9 +99,10 @@ const gaAudienceProxyRequest = async request => {
   );
   if (
     !firstResponse.success &&
-    !isHttpStatusSuccess(firstResponse.response.status)
-  )
+    !isHttpStatusSuccess(firstResponse.response.response.status)
```

Comment:
Is this correct how was this working earlier?
@ItsSudip @akashrpo

Reply:
It was not working earlier. Code coverage was not here

Reply:
I mean didn't we do negative test cases then. Actual integration tests to destination with incorrect parameters to mimick these failure case. @ItsSudip ?

Reply:
```isHttpStatusSuccess``` is getting undefined and returning false, so it was working.